tests/test_cli.py

```diff
@@ -2,19 +2,33 @@
 
 import pytest
 import sh
-
+from typing import Optional
 import dotenv
 from dotenv.cli import cli as dotenv_cli
 from dotenv.version import __version__
 
 
-def test_list(cli, dotenv_file):
+@pytest.mark.parametrize(
+    "format,content,expected",
+    (
+        (None, "x='a b c'", '''x=a b c\n'''),
+        ("simple", "x='a b c'", '''x=a b c\n'''),
+        ("json", "x='a b c'", '''{\n  "x": "a b c"\n}\n'''),
+        ("shell", "x='a b c'", '''x='a b c'\n'''),
+        ("export", "x='a b c'", '''export x='a b c'\n'''),
```

Comment:
It would be great if these have test cases specific to using `shlex.quote`. Otherwise, If there is no difference between `shell` and `simple` options, we can consider deprecating `simple` and using `shell` as default.

Reply:
There are differences with escaping special characters. I will add test cases.

Reply:
Thank you @sammck :)

Reply:
pushed...